config/database.yml

```diff
@@ -19,6 +19,7 @@ default: &default
   encoding: unicode
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
+  username: postgres
```

Comment:
It would be good to exclude this change which seems specific to @lenikadali's development environment.

Reply:
Whoops! My bad 🙃 I must have used `git commit -a` when committing, forgetting that this wasn't something I wanted to push upstream. Will remove.